### PR TITLE
Soc

### PR DESCRIPTION
**Pull request by rohitkori**

### CODE REVIEW

File: src/woc/settings.py [hunk 1]
```diff
@@ -177,8 +177,8 @@
 REST_FRAMEWORK = {
     'DEFAULT_FILTER_BACKENDS': ['django_filters.rest_framework.DjangoFilterBackend'],
     'DEFAULT_PERMISSION_CLASSES': (
-    # 'rest_framework.permissions.AllowAny',
-    'rest_framework.permissions.IsAuthenticated'
+    'rest_framework.permissions.AllowAny',
+    # 'rest_framework.permissions.IsAuthenticated',
```
Comment (RahulGopathi) [inline src/woc/settings.py:181]: Redundant changes

File: ui/src/components/CreateMentorProfile.vue [hunk 2]
```diff
@@ -124,6 +128,7 @@
       }
     },
     mounted () {
+      console.log(localStorage)
```
Comment (RahulGopathi) [inline ui/src/components/CreateMentorProfile.vue:131]: remove all the console logs

File: ui/src/router/middlewares/auth.js [hunk 3]
```diff
@@ -2,17 +2,25 @@ import store from '../../store'
...
-      next()
     }
+    // if (!store.getters['auth/isLoggedIn'] && localStorage.getItem('loginStatus')) {
+    //   next({
+    //     path: '/sign-in',
+    //     query: { next: to.fullPath }
+    //   })
+    // } else {
+    //   next()
+    // }
```
Comment (RahulGopathi) [inline ui/src/router/middlewares/auth.js:20]: remove all these comments

**Review by RahulGopathi — CHANGES_REQUESTED** [review]

**Review by RahulGopathi — APPROVED** [review]

File: ui/src/router/middlewares/auth.js (hunk 3 above)
Comment (rohitkori) [reply ui/src/router/middlewares/auth.js]: done

File: ui/src/components/CreateMentorProfile.vue (hunk 2 above)
Comment (rohitkori) [reply ui/src/components/CreateMentorProfile.vue]: done

**Review by RahulGopathi — APPROVED** [review]

**Pull request merged**